jfrog-applications/jfrog-cli/cli-command-summaries.md

```diff
@@ -0,0 +1,117 @@
+# CLI Command Summaries
+
+## Overview
+
+The command summaries feature allows the recording of JFrog CLI command outputs into the local file system. This can be used to display a summary of the executed CLI command.
+
+Each command execution that has implemented this feature can save data files into the file system. These files will then be used to create a summary markdown.
```

Comment:
* Let's share here also the list of commands that currently support markdown summary aggregation. 
* Let's also reference the setup-jfrog-cli as an example for a component that uses this feature and add a link to its documentation section that shows the Job Summary.
* Also, I suggest we use the term "markdown summary aggregation" consistently throughout this documentation page, and also in the page title.